db/db.js

```diff
@@ -294,6 +294,52 @@ async function updateSessionStillCounter(stillcounter, sessionid, locationid) {
   }
 }
 
+async function isOverdoseSuspectedInnosent(session, location) {
+  const now = new Date()
```

Comment:
What do you think about adding a DB call to get the database's time like @risatronic did here (https://github.com/bravetechnologycoop/BraveButtons/pull/88/files#diff-0e0c962c17d2dbb509faf124ae28f7a93341ee047ca1ebace5e1d0ee6f9d831eR846) and using it to get the value of `now`? In general, it can be risky to compare times that were generated on different servers. If you use the DB call, then both the "current time" and the `session.start_time` would have come from the same place.